SecurityPkg/VariableAuthenticated/SecureBootConfigDxe/SecureBootConfigImpl.c

```diff
@@ -2539,16 +2540,24 @@ UpdateDeletePage (
   EFI_SIGNATURE_LIST  *CertList;
   EFI_SIGNATURE_DATA  *Cert;
   UINT32              ItemDataSize;
-  CHAR16              *GuidStr;
-  EFI_STRING_ID       GuidID;
+  CHAR16              *CertificateInfoStr;
+  CHAR8               *CertificateInfoStr8;
+  EFI_STRING_ID       CertificateInfoID;
   EFI_STRING_ID       Help;
+  UINTN CertificateInfoStrSize;
+  CHAR16* UNKNOWN_CERT = L"Unknown Certificate: No Common Name, No Issuer";
+  RETURN_STATUS       RStatus;
 
-  Data              = NULL;
-  CertList          = NULL;
-  Cert              = NULL;
-  GuidStr           = NULL;
-  StartOpCodeHandle = NULL;
-  EndOpCodeHandle   = NULL;
+
+  Data                = NULL;
+  CertList            = NULL;
+  Cert                = NULL;
+  CertificateInfoStr  = NULL;
+  CertificateInfoStr8 = NULL;
+  StartOpCodeHandle   = NULL;
+  EndOpCodeHandle     = NULL;
+  CertificateInfoStrSize = 0;
+  RStatus = RETURN_SUCCESS;

```

Comment:
```suggestion
  Data                   = NULL;
  CertList               = NULL;
  Cert                   = NULL;
  CertificateInfoStr     = NULL;
  CertificateInfoStr8    = NULL;
  StartOpCodeHandle      = NULL;
  EndOpCodeHandle        = NULL;
  CertificateInfoStrSize = 0;
  RStatus                = RETURN_SUCCESS;
```

Keep everything aligned.

Reply:
[Aligned](https://github.com/Dasharo/edk2/compare/f2baf9f60b923e548f716217d022975fad0258d5..b60143bf81544810caea7ea1c67add0921bd80ff#diff-49fdb815b38763cce7b30e2aea02c17111bc51206950ff1d21e0073ebd0cb1d3R2551)